src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -121,4 +142,8 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws ConduitAPIException, IOException {
         return conduit.perform(methodName, params);
     }
+
+    protected JSONObject callConduit(String methodName, JSONObject params) throws ConduitAPIException, IOException {
```

Comment:
Put another way, does it make sense for the call sites to construct the JSON themselves? A Map seems conceptually cleaner to me since it decouples the eventually serialization format from the method signature.

It looks to me like the main advantage is to using the builder (`.element()`) methods above, but I'm wondering if that convenience is worth the overhead of now having two different ways to call conduit (even though the implementation is quite lean).